ios-xcframework/Podfile.lock

```diff
@@ -676,4 +676,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 1caec69a3de9c6e37113c8eb5dbdb2d2b52c5108
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.13.0
```

Comment:
Was this change needed? I see we use `1.12.1` in [WordPress iOS](https://github.com/wordpress-mobile/WordPress-iOS/blob/bd143e2aa4664db419779de92162d6b8514c4028/Podfile.lock#L246C12-L246C18)

Reply:
This change was the output of running `bundle exec pod install` (which was a command prompted by CI failures). 1.13.0 refers to the latest [Cocoapods release](https://github.com/CocoaPods/CocoaPods/releases), and 1.12.1 was the previous version. I'm not certain if they need to be consistent with WordPress iOS either.

@mokagio Any issues with using fastlane-plugin-wpmreleasetoolkit 9.1.0 or Cocoapods 1.13.0 here? Or should we use 9.0 and 1.12.1?

Reply:
It's okay to use 1.13.0 here. We don't need to keep the versions in sync between projects because:

1. Fastlane is used for project automation, there is no information exchanged between the two setup.
2. CocoaPods is used to put together dependencies and generate a binary framework. It's highly unlikely that a difference in the library version will result in issues.

Anyways, we'll soon update CocoaPods to version 1.13.0 in WordPress iOS, too, given it has [a fix for Xcode 15](https://github.com/CocoaPods/CocoaPods/releases/tag/1.13.0).